.github/workflows/acceptance.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        filter: [ api, erc20, htsprecompile, htsprecompilev1 ]
+        filter: [ api, erc20, htsprecompile, htsprecompilev1, htsprecompilev2 ]
```

Comment:
Please use a different tag.
Precompile version are being used to catch when `IHederaTokenService` contract when through a breaking change